ngshare_exchange/fetch_feedback.py

```diff
@@ -1,42 +1,19 @@
 #!/usr/bin/python
 import os
-import shutil
 import glob
+from pathlib import Path
 
 from nbgrader.exchange.abc import (
     ExchangeFetchFeedback as ABCExchangeFetchFeedback,
 )
 from .exchange import Exchange
 
-from nbgrader.utils import (
-    check_mode,
-    notebook_hash,
-    make_unique_key,
-    get_username,
-)
-from nbgrader.utils import parse_utc
-
 
 class ExchangeFetchFeedback(Exchange, ABCExchangeFetchFeedback):
     def init_src(self):
         if self.coursedir.course_id == '':
             self.fail('No course id specified. Re-run with --course flag.')
 
-        if self.coursedir.student_id != '*':
```

Comment:
Why is this removed?

Reply:
The purpose of this block is to set `student_id`, but the variable is never used.